core-java-modules/core-java-streams-4/src/test/java/splitting/Article.java

```diff
@@ -0,0 +1,11 @@
+package splitting;
+
+public class Article {
+    String target;
+    boolean featured;
```

Comment:
Should probably be `private`

Reply:
And let's add getters to access the fields

Reply:
Done.